src/components/dialogs/AddStaffNotationDialog.vue

```diff
@@ -149,7 +149,8 @@ export default class AddStaffNotationDialog extends Mixins(DateMixin, EnumMixin)
   private get notationRules (): Array<Function> {
     if (this.enableValidation) {
       return [
-        (v: string) => !!v || `Enter a ${this.displayName}`,
+        (v: string) => !!v || ((this.administrativeDissolution || this.putBackOn) ? 'Enter a detailed comment'
+          : `Enter a ${this.displayName}`),
```

Comment:
Could you please add/update a unit test that verifies the validation on this field depending on conditions? Thanks!

Reply:
done

Reply:
Beautiful!

Ready to merge unless you want Faiz to review this.